rec and auth: Implement an cors flag to set allowed origin in webserver (#17297)
PowerDNS/pdns: docs/settings.rst

If the webserver should print arguments.

.. _setting-webserver-allow-cross-origin-requests:

> ``webserver-allow-cross-origin-requests``

Reviewer: Maybe change the name into something without "allow" now?

```suggestion
``webserver-cross-origin-request-header``
```
or something like that...

Author: You make too much sense. :)

Reviewer: I'm sorry. This happens sometimes, probably as part of a curse or something.

> ``webserver-cross-origin-request_header``

Reviewer: ```suggestion
``webserver-cross-origin-request-header``
```

-----------------------------------------
.. versionadded:: 5.1.0

-  String
-  Default: empty

> The value if the access-control-allow-origin HTTP header to include. This header is not inluded if the value is empty.

Reviewer: ```suggestion
The value of the access-control-allow-origin HTTP header to include. This header will not be included if the value is empty.
```
